spec/controllers/affiliates_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
   render_views
 
   before(:all) { create_users }
-
+  before(:each) { @authable = create(:facility) }
```

Comment:
It's part of 202f349d8262a2a2f0c414996754567ec1f84aae. Specs like this using `controller_spec_helper.rb` indirectly create Facility Staff user roles, and if `@authable` isn't defined, the `UserRole`s are created without a Facility. These would no longer pass validation, and the specs fail.